apps/frontend/src/libs/components/quiz-categories-selection/libs/hooks/use-quiz-categories.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from "react";
+
+import { DataStatus } from "~/libs/enums/enums.js";
+import { useAppDispatch, useAppSelector } from "~/libs/hooks/hooks.js";
+import {
+	actions as quizCategoriesActions,
+	type QuizCategoryDto,
+} from "~/modules/quiz-categories/quiz-categories.js";
+
+const useQuizCategories = (): {
+	isLoading: boolean;
+	quizCategories: QuizCategoryDto[];
+} => {
+	const dispatch = useAppDispatch();
+
+	useEffect(() => {
+		dispatch(quizCategoriesActions.fetchQuizCategories()).catch(() => {});
```

Comment:
```suggestion
		dispatch(quizCategoriesActions.fetchQuizCategories());
```

Reply:
Got this lint error when I deleted the catch method.

![Screenshot (336)](https://github.com/user-attachments/assets/0d28e672-67a4-4ccd-b111-31ac0e3bcd7a)

Reply:
My bad, already solved it with `void` keyword